src/App.tsx

```diff
@@ -38,75 +40,156 @@ const hederaTestnet = {
   },
 } as const satisfies Chain;
 
+// The chains used in the application
 const chains = [hederaTestnet];
-const projectId = "58bc91d22471456becc991ee47b66d22";
 
+// The projectId for the WalletConnect integration
+const projectId = "bc8a48da849089da783de2987986630f";
+
+// The public client used in the application
 const { publicClient } = configureChains(chains, [w3mProvider({ projectId })]);
 
+// The wagmi configuration used in the application
 const wagmiConfig = createConfig({
   autoConnect: true,
   connectors: w3mConnectors({ projectId, chains }),
   publicClient,
 });
 
+// The ethereum client used in the application
 const ethereumClient = new EthereumClient(wagmiConfig, chains);
 
-const mintableTokenContractAddress =
-  "0x0000000000000000000000000000000000edaabf";
+// Variable used to store the token id
+let tokenId;
 
+// Variables used to store the provider and signer
 let provider: ethers.BrowserProvider;
 let signer: ethers.Signer;
 
+// Function used to initialize the provider and signer variables
 async function initProvider() {
   provider = new ethers.BrowserProvider(window.ethereum);
   signer = await provider.getSigner();
 }
 
-async function transferHbar() {
+// Function used to create a fungible token using the SDK
+async function createToken() {
+  console.log("creating token...")
+  tokenId = await createFungibleToken("Mintable Token", "MT");
+  alert("Token created successfully with id: " + tokenId);
+}
+
+// Function used to associate a new account with the token using the smart contract
+async function associateAccountWithToken() {
+
   if (!provider) initProvider();
 
-  const response = await provider.send("eth_chainId", []);
+  // Get the signer
+  signer = await provider.getSigner();
+
+  try {
+    // Get the solidity address from the token id
+    const mintableTokenContractAddress = '0x' + tokenId!.toSolidityAddress();
+
+    // Create the contract instance to interact with the smart contract
+    const mintableTokenContract = new ethers.Contract(
+      mintableTokenContractAddress,
+      mintableTokenAbi,
+      signer,
+    );
+
+    // Associate the new account with the token using the smart contract
```

Comment:
`Associate the new account with the token using the smart contract` => `Associate the new token with the current Metamask account using the contract instance`